config/stack_mac.yaml

```diff
@@ -71,7 +71,7 @@ pio:
 
 esmf:
   build: YES
-  version: 8_1_0_beta_snapshot_27
+  version: 8_1_0_beta_snapshot_21
   shared: YES
```

Comment:
Can you remind me why are we building the shared library at all?

Reply:
I don't have a good answer. Because shared libraries are the standard on macOS? JEDI related? Maybe @aerorahul has an answer.

Reply:
On a macOS. Yes. On NOAA platforms it is static. 
I have no preference. Just want to test shared somewhere. I prefer shared, in general. And seems macOS is a niche club.

Reply:
I usually build the static library only, it's cleaner and you avoid potential `rpath` issues on macOS.